CONTRIBUTING.md

```diff
@@ -1,31 +1,30 @@
-# Contributing
+# CONTRIBUTING
 
-## Important notes
```

Comment:
I'd rather use the stock CONTRIBUTING file, I'm fine with leaving out the common sense stuff.